docs/components/plugins/api/APISectionUtils.tsx

```diff
@@ -492,23 +611,54 @@ export const STYLES_SECONDARY = css`
 `;
 
 export const STYLES_PLATFORM = css`
-  & {
-    display: inline-block;
-    background-color: ${theme.background.tertiary};
-    color: ${theme.text.default};
-    font-size: 90%;
-    font-weight: 700;
-    padding: 6px 12px;
-    margin-bottom: 8px;
-    margin-right: 8px;
-    border-radius: 4px;
-  }
+  display: inline-block;
+  background-color: ${theme.background.tertiary};
+  color: ${theme.text.default};
+  font-size: 90%;
```

Comment:
Instead of manually defining type styles, let's use our text components. 

This one can probably be `<CALLOUT>` or `<FOOTNOTE>`. Those components have responsiveness built in and will be consistent with the website.

Reply:
I would wait until we ship the redesign with it, as stated before many component will need to be replaced, IIRC also CALLOUT and FOOTNOTE components you are referring to.